src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -177,15 +173,15 @@ ApplicationManagerImpl::ApplicationManagerImpl(
     , is_low_voltage_(false)
     , apps_size_(0)
     , is_stopping_(false) {
-  std::srand(std::time(nullptr));
+  std::srand(std::time(0));
   AddPolicyObserver(this);
 
   dir_type_to_string_map_ = {{TYPE_STORAGE, "Storage"},
                              {TYPE_SYSTEM, "System"},
                              {TYPE_ICONS, "Icons"}};
 
   sync_primitives::AutoLock lock(timer_pool_lock_);
-  TimerSPtr clearing_timer(std::make_shared<timer::Timer>(
+  TimerSPtr clearing_timer(utils::MakeShared<timer::Timer>(
```

Comment:
@Ypostolov Check all these changes. We already use shared_ptr from C++11 . Return previous code please